simapp/simd/cmd/root.go

```diff
@@ -75,6 +78,45 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// initAppConfig helps to override default appConfig template and configs.
+// return "", nil if no custom configuration is required for the application.
+func initAppConfig() (string, interface{}) {
```

Comment:
@anilCSE do you think this could be reused to override tendermint config.toml?

Reply:
I believe we can do that. I will look into it tomorrow and create a PR

Reply:
![](https://media2.giphy.com/media/S5ciiTMG3U1Qrt1IQ6/giphy.gif)